test/example.c

```diff
@@ -923,11 +923,18 @@ void test_deflate_set_header(unsigned char *compr, size_t comprLen) {
     CHECK_ERR(err, "deflateInit2");
 
     head->text = 1;
+    head->comment = "comment";
+    head->name = "name";
+    head->hcrc = 1;
+    head->extra = "extra";
+    head->extra_len = strlen(head->extra);
+
     err = PREFIX(deflateSetHeader)(&c_stream, head);
     CHECK_ERR(err, "deflateSetHeader");
     if (err == Z_OK) {
         printf("deflateSetHeader(): OK\n");
     }
+    PREFIX(deflateBound)(&c_stream, comprLen);
```

Comment:
This does nothing... It gets optimized away...

Reply:
Are you sure? Its getting covered by code coverage..

Reply:
Because the return value is ignored, it gets optimized away.